src/main/java/org/apache/commons/lang3/math/NumberUtils.java

```diff
@@ -424,42 +426,42 @@ public static Number createNumber(final String str) {
                         // ignore the bad number
                     }
                     //$FALL-THROUGH$
-                case 'd' :
-                case 'D' :
-                    try {
+                case 'd' :  // + 1
+                case 'D' :  // + 1
+                    try {    // + 1
                         final Double d = createDouble(str);
-                        if (!(d.isInfinite() || d.doubleValue() == 0.0D && !isZero(mant, dec))) {
+                        if (!(d.isInfinite() || d.doubleValue() == 0.0D && !isZero(mant, dec))) {   // + 3
                             return d;
                         }
                     } catch (final NumberFormatException ignored) {
                         // ignore the bad number
                     }
-                    try {
+                    try {  // + 1
                         return createBigDecimal(numeric);
                     } catch (final NumberFormatException ignored) {
                         // ignore the bad number
                     }
                     //$FALL-THROUGH$
-                default :
+                default :   // + 1
```

Comment:
I'm not entirely certain, but it's possible that the "default" option may not be included in the count.